tests/integration/test_pip.py

```diff
@@ -96,6 +96,30 @@
             ),
             id="pip_yanked",
         ),
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/cachi2-pip-wheels.git",
+                ref="3697d63affaec82985f1d9b4035d5305e58c91d6",
+                packages=({"path": ".", "type": "pip", "allow_binary": "true"},),
+                check_vendor_checksums=False,
+                expected_exit_code=0,
+                expected_output="All dependencies fetched successfully",
+            ),
+            id="pip_no_wheels",
+        ),
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/cachi2-pip-wheels.git",
+                ref="eee59273542ee7d412fb359d471386b645cf166e",
```

Comment:
This is the same branch as for e2e test. It fails on pre-fetching `tensorboard` as intended.